src/controller/osw-controller.ts

```diff
@@ -384,6 +385,48 @@ class GtfsOSWController implements IController {
 
         }
     }
+    /**
+     * Gives the downloadable stream for the job status
+     * @param request 
+     * @param response 
+     * @param next 
+     * @returns 
+     */
+    getFormatDownloadFile = async (request: Request, response: express.Response, next: NextFunction) => {
+
+        console.log('Download formatted file for jobInfo ')
+        try {
+            const job_id = request.params['job_id'];
+            if (job_id == undefined || job_id == '') {
+                return next(new InputException('job_id not provided'));
+            }
+            const jobInfo = await oswService.getOSWFormatJob(job_id);
+
+            if (jobInfo.status != 'completed'){
+                throw new JobIncompleteException(job_id);
+            }
+            // Get the file entity for the file
+            const fileEntity = await oswService.getFileEntity(jobInfo.target_url);
+            if (jobInfo.target == 'osm') {
+                // OSM implies xml file
+            response.setHeader('Content-Type', 'application/xml');
+            response.setHeader('Content-Disposition', `attachment; filename=${fileEntity.fileName}`);
+            (await fileEntity.getStream()).pipe(response);
+
+            }
+            else if (jobInfo.target == 'osw') {
+                response.setHeader('Content-Type', 'application/zip');
```

Comment:
put if else only for content type

Reply:
there is an else that throws error. Cannot make this change